scene/gui/color_picker.h

```diff
@@ -93,14 +120,19 @@ class ColorPicker : public BoxContainer {
 	HSeparator *preset_separator = memnew(HSeparator);
 	Button *btn_add_preset = memnew(Button);
 	Button *btn_pick = memnew(Button);
-	CheckButton *btn_hsv = memnew(CheckButton);
-	CheckButton *btn_raw = memnew(CheckButton);
-	HSlider *scroll[4];
-	SpinBox *values[4];
-	Label *labels[4];
+
+	OptionButton *mode_option_button = memnew(OptionButton);
+
+	HSlider *scrolls[SLIDER_COUNT];
```

Comment:
```suggestion
	HSlider *scrolls[SLIDER_COUNT] = nullptr;
```
No idea if this will work; raw arrays like this are rarely used in the codebase.